frontend/app/src/background.ts

```diff
@@ -5,10 +5,8 @@ import { ipcSetup } from '@/electron-main/ipc-setup';
 import { getUserMenu } from '@/electron-main/menu';
 import { TrayManager } from '@/electron-main/tray-manager';
 import { type Nullable } from '@/types';
-import { checkIfDevelopment } from '@/utils/env-utils';
```

Comment:
can you try removing the contents of the dist folder and running dev again? for me it seems that at least for this the auto-import doesn't work and background gives an error.

Reply:
so after much digging, I found out that, we have 3 vite config files, which run separately.
we only have auto imports setup for the web app dev/build, we do not have it on electron dev/build vite config files.

so, I extracted the config and re-used it on all three vite runs.